src/cmds/proc/thread.c

```diff
@@ -29,22 +30,24 @@ static void print_stat(void) {
 	int total;
 	struct task *task;
 
-	printf(" %4s %4s %8s %7s %10s\n", "id", "tid", "priority", "state", "time");
+	printf(" %4s %16s %8s %7s %10s %12s\n", "id", "tid/Task name", "priority", "state", "time", "Stack size");
 
 	running = sleeping = suspended = 0;
 
 	sched_lock();
 	{
 		task_foreach(task) {
 			task_foreach_thread(t, task) {
-				printf(" %4d %4d %8d %c %c %c %c %9lds\n",
+				printf(" %4d %4d/%-11s %8d %c %c %c %c %9lds %9zu\n",
```

Comment:
what is difference between `%-11s` and `%11s`

Reply:
Its just making the printf look good.
%11s  does it as : Task_Name__
%-11s does it as : __Task_Name

where '_' denotes white space.